pkg/aws/policies.go

```diff
@@ -210,6 +213,52 @@ func (c *awsClient) AttachRolePolicy(roleName string, policyARN string) error {
 	return nil
 }
 
+func (c *awsClient) FindPolicyARN(operator Operator, version string) (string, error) {
+	listPoliciesOutput, err := c.iamClient.ListPolicies(&iam.ListPoliciesInput{
```

Comment:
I see that we're applying a scope here but larger organizations might have > 100 policies. We might want to use `[ListPoliciesPagesWithContext](https://docs.aws.amazon.com/sdk-for-go/api/service/iam/#IAM.ListPoliciesPagesWithContext)` instead to paginate.